pkg/overlay/cache.go

```diff
@@ -47,6 +47,9 @@ type DB interface {
 	Get(ctx context.Context, nodeID storj.NodeID) (*NodeDossier, error)
 	// GetAll looks up nodes based on the ids from the overlay cache
 	GetAll(ctx context.Context, nodeIDs storj.NodeIDList) ([]*NodeDossier, error)
+
+	// UnhealthyOrOffline filters a set of nodes to unhealth or offlines node, independent of new
+	UnhealthyOrOffline(context.Context, *NodeCriteria, storj.NodeIDList) (storj.NodeIDList, error)
```

Comment:
`Unhealthy` should be sufficient for the method name.

Reply:
I was trying to establish strong definitions between online / healthy / new because the old definitions left logical gaps.

Reply:
Ah, ok... Wonder whether we should have a separate name for those nodes. e.g. "Problematic", But... yeah, `UnhealthyOrOffline` makes sense in that case.

Reply:
It was 'invalid', but that seemed to vague.  We briefly used (not) 'vetted' per the white paper until I realized this was different.  I'd like to have some word with a clear antonym.  Stable / unstable?

Reply:
`Unstable`, `Unreliable`, `Unsafe`, `Undependable`, `Unsound` (see antonyms in https://www.thesaurus.com/browse/stable?s=t)

Maybe `Unreliable` or `Unsafe`?